### ISSUE TITLE
Support generic file structures

### ISSUE THREAD

**Issue by JMBeresford** [JMBeresford/retrom#104]
The current restriction on file structure of a user's library is affecting user adoption of Retrom. Retrom should support arbitrary file/directory structures for a library, and allow a user to define where their entries live via some config.

- [x] Single File Game Support #108 
- [ ] Single Platform Support

**Comment by Professorvennie** [comment]
I would love to switch to retrom from romm but I am waiting on the folder suppport.  Any update on this issue?

**Comment by JMBeresford** [comment]
@Professorvennie specifically what folder structure are you referring to? Retrom now supports single-file games in addition to the originally supported multi-file games ( [see README](https://github.com/JMBeresford/retrom#library-structure) for illustrations of each ).

Generic directory structure support is still incoming. I can only guess as to an ETA, but I imagine within a month or so.

I'm always open to contributions as well, for anyone that can't wait fr me to get to it myself 😃

**Comment by Jerrys-modz** [comment]
@JMBeresford yes I just can’t read I guess 😅. Just spun up a container! Thank you for the great work!
